src/appmixer/servicenow/routes.js

```diff
@@ -0,0 +1,27 @@
+module.exports = async context => {
+
+    context.http.router.register({
+        method: 'POST',
+        path: '/events',
+        options: {
+            auth: false,
+            handler: async req => {
+
+                const { data = {}, type } = req.payload || {};
+
+                if (!type) {
+                    context.log('error', 'Missing type property.', req.payload);
```

Comment:
This gives me an error:
![image](https://github.com/user-attachments/assets/44bc0ece-1268-4065-8d3f-213612d5d1a8)

```suggestion
                    context.log({ step: 'error', message: 'Missing type property.', payload: req.payload});
```